openstf/model/confidence_interval_applicator.py

```diff
@@ -7,19 +7,29 @@
 import numpy as np
 import pandas as pd
 from scipy import stats
+from sklearn.base import RegressorMixin
+
+from openstf.enums import MLModelType
 
 MINIMAL_RESOLUTION: int = 15  # Minimal time resolution in minutes
 
 
 class ConfidenceIntervalApplicator:
-    def __init__(self, model):
-        self.confidence_interval = model.confidence_interval
+    def __init__(self, model: RegressorMixin, forecast_input_data: pd.DataFrame):
+        self.model = model
+        self.forecast_input_data = forecast_input_data
 
-    def add_confidence_interval(
-        self, forecast: pd.DataFrame, quantiles: list
-    ) -> pd.DataFrame:
+    def add_confidence_interval(self, forecast: pd.DataFrame, pj: dict) -> pd.DataFrame:
         temp_forecast = self._add_standard_deviation_to_forecast(forecast)
```

Comment:
Een docstring over wat deze functie doet en wat de output is (welke kolommen bijv) zou mega chill zijn:)